webapp/apps/btax/compute.py

```diff
@@ -0,0 +1,57 @@
+import dropq
+import os
+from ..taxbrain.models import WorkerNodesCounter
+import json
+import requests
+from requests.exceptions import Timeout, RequestException
+from .helpers import arrange_totals_by_row
+from ..taxbrain.compute import (DropqCompute,
+                                MockCompute,
+                                MockFailedCompute,
+                                NodeDownCompute,
+                                JobFailError,
+                                ENFORCE_REMOTE_VERSION_CHECK,
+                                TIMEOUT_IN_SECONDS,
+                                dropq_version)
+import requests_mock
+requests_mock.Mocker.TEST_PREFIX = 'dropq'
+
+
+
+def package_up_vars(self, user_mods, first_budget_year):
```

Comment:
`package_up_vars` is added to the classes below that inherit from taxbrain/compute.py classes and for that reason needs `self` as first arg.  I realize it looks a little odd.  I tried a more traditional inheritance pattern and was getting a [method resolution error](https://travis-ci.org/OpenSourcePolicyCenter/webapp-public/builds/152743982#L733) I had never seen before.